tetris/board.go

```diff
@@ -41,12 +46,17 @@ func (b *Board) Emprint(piece Piece) (int, bool) {
 
 	for _, p := range piece.points {
 		b.m[p.x][p.y] = 1
+		b.pieceColors[p.x][p.y] = piece.GetColor()
 	}
 
 	rmCnt := b.removeFillRows()
 	return rmCnt, true
 }
 
+func (b *Board) GetPieceColors() [][]string {
```

Comment:
The name of the function should be PieceColors, without Get. We don't want Get / Sets in Go.